test/consts.test.ts

```diff
@@ -36,7 +36,7 @@ describe('deprecated contract addresses', () => {
       '0x6884d88Ce56C5C93F46eE23684eBA8628c90B518'
     );
     expect(SYLVESTER_POLYGON_ADDRESS).to.equal(
-      '0xfA06cFE34C85Ec6b6D29A6a99806cC68BA0018Fe'
```

Comment:
These were held onto for backwards compatibility with the old SDK, before the release became a major version bump. Consumers of that SDK would expect `SYLVESTER_POLYGON_ADDRESS` to resolve to the `v0` deployment.

Since we've rolled out a new version, and it's an antipattern to avoid using a `RenftContractDeployment`, shall we remove all of these global declarations altogether? (This won't invalidate the application frontend since we don't use any of them.)

Reply:
Raised: https://linear.app/renft/issue/NFT-1083/remove-exported-address-force-callers-to-use-finddeployments